src/shared/nt-products/ProductListPage.styl

```diff
@@ -0,0 +1,50 @@
+@require('../nt-design/colorsets')
+@require('../nt-styles/global')
+
+.column
+    padding 0.75rem
+ 
+    &__half
+        -webkit-box-flex 0
+        -ms-flex none
+        flex none
+        width 50%
+
+    &__offset-one-quarter
+        margin-left 25%
+
+.panel
+    font-size 1rem
+
+    &__head
+        background-color whitesmoke
+        border-radius 3px 3px 0 0
+        color #363636
+        font-size 1.25em
+        font-weight 300
+        line-height 1.25
+        padding 10px 25px
+
+        &:first-child
+            border-top 1px solid c-gray300
+    &__block
+        cursor pointer
+        -webkit-box-align center
+        -ms-flex-align center
+        align-items center
+        color #363636
+        display -webkit-box
+        display -ms-flexbox
+        display flex
+        -webkit-box-pack start
+        -ms-flex-pack start
+        justify-content flex-start
```

Comment:
We are using `autoprefixer` so `-webkit` `-ms` will be automatically added. You don't need to add it by yourself.
Anyways, I didn't add the `autoprefixer` in the dev env yet you can add it.